report.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto2";
+import "google/protobuf/timestamp.proto";
+import "entity.proto";
+
+enum ReportType {
+  REPORT_TYPE_UNDEFINED = 0;
+  REPORT_TYPE_DRIVER_REPORT = 1;
+}
+enum ReportRole {
+  REPORT_ROLE_UNDEFINED = 0;
+  REPORT_ROLE_BEFORE_SERVICE = 1;
+  REPORT_ROLE_AFTER_SERVICE = 2;
+}
+
+message Report {
+  required uint64 id = 1;
+  required google.protobuf.Timestamp created_at = 2;
+  required string name = 3;
+  required ReportType type = 4;
+  required ReportRole role = 5;
+  required uint64 organization_id = 6;
+  required string storage_path = 7;
+  required uint64 author_entity_id = 8;
+  optional uint64 location_id = 9;
+  optional uint64 process_id = 10;
+  required EntityStatus status = 11;
+  required string bucket = 12;
```

Comment:
are you emphasizing that the URL to the picture will be passed, right?

Reply:
because currently, I setup permissions that way, that only backend pods can access ObjectStorage

Reply:
yes, the assumption is that files can be accessed by full path. Since the filename is `uuid.uuid4()`, this was considered ok`ish.

If we don't provide permissions, then backend will need to download image from object storage and send it to the user. I wanted to avoid that. Do I miss smth? 🤔🤷‍♂️

Reply:
If it accessible via url directly, it would basically become publicly accessible